frontend/src/common/queries/filter.ts

```diff
@@ -506,7 +502,7 @@ export function buildSummaryCitation(
   publisherMetadata?: PublisherMetadata
 ): string {
   if (!publisherMetadata) {
-    return "";
+    return CATEGORY_VALUE_KEY.NO_PUBLICATION;
```

Comment:
"No value" value is now `NO_PUBLICATION`. [See comment above](https://github.com/chanzuckerberg/single-cell-data-portal/pull/5678/files#r1312136599).